webextensions/api/tabs.json

```
@@ -4153,7 +4146,7 @@
               "description": "`updateProperties.selected` parameter",
               "support": {
                 "chrome": {
-                  "version_added": "≤58"
+                  "version_added": "18"
```

:x: Same as `update.updateProperties_pinned_parameter`:

```suggestion
                  "version_added": "9"
```